packages/openapi-v3/src/__tests__/unit/enhancers/consolidate-schema.enhancer.unit.ts

```diff
@@ -0,0 +1,197 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/openapi-v3
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+import {ConsolidationEnhancer, OpenAPIObject} from '../../..';
+
+const consolidationEnhancer = new ConsolidationEnhancer();
+
+describe('consolidateSchemaObjects', () => {
+  it('moves schema with title to component.schemas, replace with reference', () => {
+    const inputSpec: OpenAPIObject = {
+      openapi: '',
+      info: {
+        title: '',
+        version: '',
+      },
```

Comment:
Please use `OpenApiSpec` builder to avoid dealing with boilerplate required to create a valid spec. See https://github.com/strongloop/loopback-next/tree/master/packages/openapi-spec-builder.

The same comment applies to other similar places in this pull request too.